<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lifeRestoreProgress</class>
 <widget class="QDialog" name="lifeRestoreProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Restoring</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lPreserve.qrc">
    <normaloff>:/images/images/lifepreserver.png</normaloff>:/images/images/lifepreserver.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelRestoreStatus">
     <property name="text">
      <string>Restore in progress... Please wait...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="viewRestoreOutput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushDone">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lPreserve.qrc"/>
 </resources>
 <connections/>
</ui>
